frontend/components/PerceivedDemographics.jsx

```diff
@@ -44,13 +44,13 @@ class PerceivedDemographics extends Component {
 
   // Prepare the annotations into the format expected by the action/reducer
   prepareAnnotationsObject() {
-    const demographics = this.props.questionOrder.map(questionName => ({
+    const annotations = this.props.questionOrder.map(questionName => ({
       name: questionName,
-      option: this.state[questionName],
+      value: this.state[questionName],
```

Comment:
This is actually what the server was expecting, so I think I was passing the wrong values up until now